pulsar-broker/src/test/java/org/apache/pulsar/broker/service/OneWayReplicatorTest.java

```diff
@@ -1665,7 +1666,16 @@ public void testReplicatorsInflightTaskListIsEmptyAfterReplicationFinished() thr
         final String subscriptionName = "s1";
         admin2.topics().createNonPartitionedTopic(topicName);
         admin2.topics().createSubscription(topicName, subscriptionName, MessageId.earliest);
-        admin1.topics().createNonPartitionedTopic(topicName);
+        try {
+            admin1.topics().createNonPartitionedTopic(topicName);
+        } catch (PulsarAdminException.ConflictException e) {
+            if (usingGlobalZK) {
+                // Since brokers shared namespace level policies, which triggers enabling a binary-way replication
+                // if using a global ZK.
+                // So the topic has been created by the replicator.
+            }
+            throw e;
```

Comment:
```suggestion
            if (!usingGlobalZK) {
                // Since brokers shared namespace level policies, which triggers enabling a binary-way replication
                // if using a global ZK.
                // So the topic has been created by the replicator.
                throw e;
            }
```